tests/testthat/test-dm.R

```diff
@@ -271,7 +269,6 @@ test_that("`pull_tbl()`-methods work", {
 
   expect_dm_error(
     new_dm3(dm_for_filter %>%
-      dm_zoom_to(t1) %>%
       dm_get_def() %>%
       mutate(zoom = list(t1)), zoomed = TRUE) %>%
```

Comment:
How does this work? Do we need instead:

```suggestion
      mutate(zoom = list(t1)), zoomed = TRUE) %>%
      new_dm3() %>%
```

Reply:
The idea is to have multiple tables at once in the `zoom`-column and your suggestion does that, and looks slightly more transparent at the same time.

Reply:
Sorry, misread, `zoomed = TRUE` is an argument for `new_dm3()`. 
I will do it like this:
```r
dm_get_def(dm_for_filter) %>%
      mutate(zoom = list(t1)) %>%
      new_dm3(zoomed = TRUE) %>%
      pull_tbl()
```